static/app/components/replays/diff/replaySliderDiff.tsx

```diff
@@ -23,21 +25,35 @@ export function ReplaySliderDiff({leftOffsetMs, replay, rightOffsetMs}: Props) {
 
   const width = toPixels(viewDimensions.width);
   return (
-    <WithPadding>
-      <Positioned ref={positionedRef}>
-        {viewDimensions.width ? (
-          <DiffSides
-            leftOffsetMs={leftOffsetMs}
-            replay={replay}
-            rightOffsetMs={rightOffsetMs}
-            viewDimensions={viewDimensions}
-            width={width}
-          />
-        ) : (
-          <div />
-        )}
-      </Positioned>
-    </WithPadding>
+    <Fragment>
+      <Header>
+        <Tooltip title={t('How the initial server-rendered page looked like.')}>
```

Comment:
```suggestion
        <Tooltip title={t('What the initial server-rendered page looked like.')}>
```
```suggestion
        <Tooltip title={t('How the initial server-rendered page looked.')}>
```
THIS IS NITTY i feel like one of these flows better

Reply:
I vote for the second one ('How the initial server-rendered page looked.')